tests/test_initial_point.py

```diff
@@ -187,6 +193,40 @@ def test_string_overrides_work(self):
         assert np.isclose(iv["B_log__"], 0)
         assert iv["C_log__"] == 0
 
+    @pytest.mark.parametrize("reverse_rvs", [False, True])
+    def test_dependent_initval_from_OFG(self, reverse_rvs):
```

Comment:
Is this test indicative of why this features is need for the other PR?

Reply:
Yup